README.md

```diff
@@ -185,26 +191,64 @@ inoremap <silent><expr> <C-e>     compe#close('<C-e>')
 - [snippets.nvim](https://github.com/norcalli/snippets.nvim)
 - [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter)
 
-
 ## FAQ
 
-#### How to use LSP snippet?
+### How to use LSP snippet?
 
 1. Set `snippetSupport=true` for LSP capabilities.
 
-```lua
-local capabilities = vim.lsp.protocol.make_client_capabilities()
-capabilities.textDocument.completion.completionItem.snippetSupport = true
+   ```lua
+   local capabilities = vim.lsp.protocol.make_client_capabilities()
+   capabilities.textDocument.completion.completionItem.snippetSupport = true
 
-require'lspconfig'.rust_analyzer.setup {
-  capabilities = capabilities,
-}
-```
+   require'lspconfig'.rust_analyzer.setup {
+     capabilities = capabilities,
+   }
+   ```
 
 2. Install `vim-vsnip`
 
-`Plug 'hrsh7th/vim-vsnip'`
+   `Plug 'hrsh7th/vim-vsnip'`
```

Comment:
Inconsistent formatting, add triple backticks here

Reply:
Good point. Changed. markdownlint did not report this one.